packages/react/src/components/DetailsList/DetailsRow.base.tsx

```diff
@@ -285,8 +288,10 @@ export class DetailsRowBase extends React.Component<IDetailsRowBaseProps, IDetai
         data-selection-index={itemIndex}
         data-selection-touch-invoke={true}
         data-item-index={itemIndex}
-        aria-rowindex={groupNestingDepth ? undefined : itemIndex + flatIndexOffset}
+        aria-rowindex={groups ? undefined : itemIndex + flatIndexOffset}
         aria-level={(groupNestingDepth && groupNestingDepth + 1) || undefined}
+        aria-posinset={ariaPositionInSet}
+        aria-setsize={groups ? groups.length : undefined}
```

Comment:
You have the same problem here, where you're setting `aria-setsize` to the number of groups in the top level, instead of the number of items in the current group. If you can find the immediate parent group instead of the top-level parent group, that would fix this issue too.